spec/support/examples/a_validation_with_correct_interface.rb

```diff
@@ -0,0 +1,7 @@
+
+# let(:validation) { SomeValidator.new }
+shared_examples_for "a validation with the correct interface" do
+  it "can successfully #perform_validation" do
+    expect { validation.perform_validation }.not_to raise_error
+  end
```

Comment:
This test will fail when the subclass incorrectly overrides `validate` and doesn't call `perform_validation` at all.